frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestionView.tsx

```diff
@@ -97,8 +98,8 @@ export function PublicOrEmbeddedQuestionView({
         {() => (
           <Visualization
             isNightMode={theme === "night"}
-            error={result && result.error}
-            rawSeries={[{ card: card, data: result && result.data }]}
+            error={result?.error?.toString()}
```

Comment:
cc @metabase/embedding 
the `error` is a `string` or an object with 2 fields.

So we stringify the object.